pkg/jobrunaggregator/jobrunaggregatorapi/types_row_release.go

```diff
@@ -48,6 +48,9 @@ type ReleaseRow struct {
 
 	// OSDiffURL is a link to the release page diffing the two OS versions.
 	OSDiffURL string `bigquery:"osDiffURL"`
+
+	// CreatedAt contains a timestamp for when this record was created in BigQuery.
+	CreatedAt time.Time `bigquery:"createdAt" json:"-"`
```

Comment:
`insertedAt` instead of `createdAt` so that even without reading docs, it's clear that this is about the insertion of the row, not the creation of a thing.

Also, the field documentation needs to warn consumers that the value isn't unique and multiple rows can be inserted with the same `insertedAt`.  In general, uploads for other tables run in parallel, so given that this is not late binding you're likely to end up with moving back in time too.